src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+import $style from './slider-improvements.module.css';
+import SliderButton from '@src/features/basic/slider-improvements/SliderButton.vue';
+import { clickElement } from '@src/util';
+
+function onTileReady(tile: PrunTile) {
+  sliderMinMaxButtons(tile);
+  sliderTableBulkMinMax(tile);
+}
+
+function sliderTableBulkMinMax(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    const headerRow = _$(table, 'thead')?.firstElementChild;
+    if (!headerRow) return;
+    const sliders = _$$(table, 'rc-slider');
+    if (sliders.length > 0) {
+      const sliderColumns = new Map<number, HTMLElement[]>();
+      sliders.forEach(slider => {
+        const cell = slider.closest('td');
+        const columnIndex = Array.prototype.slice.call(cell?.parentElement?.children).indexOf(cell);
```

Comment:
Why do you call `.slice` via `Array.prototype.slice.call` instead of calling it directly?